web/classes/UserUtil.php

```diff
@@ -264,6 +264,20 @@ public function verifySignupRequest($id, $key) {
 		
 		public function completeSignupAction($id, $key, $reonEmail, $password, $passwordConfirm) {
 			$email = self::$instance->verifySignupRequest($id, $key);
+
+			$result = self::$instance->createUser($email, $reonEmail, $password, $passwodConfirm);
```

Comment:
I moved the parts specific to creating a user to a function called `createUser` so I can call from a script